### ISSUE TITLE
Air pollution refactor to forumulas rather than direct table emissions

### ISSUE THREAD

**Issue by karlthepagan** [StrangeLoopGames/EcoSuggestions#1495]
Air pollution as implemented today is based on the table rather than the mechanics of production. As it relates to the above garbage to #1494 I think that fuel sources should gain the attributes of pollution per kilo-Joule rather than keeping pollution a feature of the table.

Mechanical and electrical generators could have abatement features which might reduce pollution per kJ.

Future smog stack world objects could treat, sequester, or abate the amount of pollution, but in general manufacturing efficiency (table upgrades) well represent the reduction of pollution as a historic analogy.

**Comment by SatsukiShizuka** [comment]
It is easy to underestimate the environmental impact of burning logs for lighting when the whole town is doing it - it's nothing that causes the sea to rise (and it shouldn't be THAT bad), but it should have some impact, yes.

Which makes ALL the more sense to use cleaner, renewable vegetable oils for lighting, albeit at a slightly higher cost.

Still waiting for the kerosene middle-step prior to electricity and all that.

**Comment by karlthepagan** [comment]
> underestimate the environmental impact of burning logs for lighting when the whole town is doing it

This is true for CO2, a campfire should have relatively low emissions. However the health impact of wood and biomass for cooking is enormous. It is the cause of millions of emphysema deaths each year in the developing world.

IMO under the current game analogy that would be a housing tier penalty. This is something I'm playing around with implementing as a part of my mod "Earthen Floor Housing".
